deployments/helm-chart/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: nginx-ingress
-version: 0.0.0-edge
-appVersion: edge
+version: 0.9.0
+appVersion: 1.11.0
 apiVersion: v1
 kubeVersion: ">= 1.16.0-0"
 description: NGINX Ingress Controller
```

Comment:
`https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v1.11.0/deployments/helm-chart/chart-icon.png` for the icon?